package.json

```diff
@@ -79,9 +78,6 @@
     "tildify": "^2.0.0",
     "tmp": "^0.2.1"
   },
-  "optionalDependencies": {
-    "ngrok": "^3.2.7"
```

Comment:
Don't we need to install it whenever it is required?

Reply:
we would dynamically install it  ref:[here](https://github.com/twilio/twilio-cli/blob/main/src/commands/phone-numbers/update.js#L38-L39) if dependency is not present and needed.

Reply:
Great. It make sense now. Thanks :)